### ISSUE TITLE
2023.18 is constalty freezing

### ISSUE THREAD

**Issue by skier150** [xLightsSequencer/xLights#4085]
Roughly every 3 minutes, Xlights freezes.  This is only happening to me with 2023.18.  No matter what I am doing, Xlights freezes, I get a blue ring for my pointer, and the screen gets an extra  white/cloudy layer.  It lasts maybe 10 seconds and then everything is back to normal, for another 3 minutes.  Nothing else running in Windows is affected, it is limited to Xlights.

![202318freez](https://github.com/smeighan/xLights/assets/149287514/838dd266-ea46-4733-a953-ca5713f95226)

Unfortunately, the screenshot does not show my pointer as a blue ring.

Running Windows 10 with the latest updates
Running Xlights 2023.18

NOTE: IF YOU DO NOT ATTACH A SEQUENCE THAT DEMONSTRATES YOUR PROBLEM THEN THERE IS A HIGH PROBABILITY YOUR ISSUE WILL JUST BE CLOSED AND NOT INVESTIGATED.

**Describe the bug**

**To Reproduce**
(and/or link to a short video showing the problem with audio track describing what you are doing)

**Expected behavior**
A clear and concise description of what you expected to happen and why.

**Screenshots**

**Versions (please complete the following information):**
 - OS: [e.g. Windows 10, Mac 10.14.5, etc...]
 - xLights version [e.g. 2019.37]

**Additional context**

**Attachments**
- If associated with a crash please attach/link to a dropbox containing the crash log that was produced
- If it crashed but no crash log was produced please use Tools Menu/Package Logs and attach/link to a dropbox containing the resulting zip file
- If the error relates to rendering an effect or a crash while sequencing please use Tools Menu/Package Sequence and attach/link to a dropbox containing the resulting zip file

If you would prefer not to post your sequences publically please say so and be ready to PM them to one of the developers when asked.

**Comment by derwin12** [comment]
Likely need more details here ... perhaps tools->package logs.
Did it do this with a previous version? Have you tried .17?

**Comment by skier150** [comment]
I am not sure if I ran .17, I think I may have gone from .16 to .18.  However, I was not having this problem with .16.  I just looked at it again, and it is pretty consistently every 3 minutes.  I thought it may be only while I was sequencing, but I went to the controller visualizer and had the same issue.  I tried loading a different sequence, no change.  I also tried it with no sequences loaded, still no change.

I am going to try and attach the logs to this reply.


    On Sunday, October 29, 2023 at 10:59:08 AM EDT, Daryl ***@***.***> wrote:  
 
 


Likely need more details here ... perhaps tools->package logs.
Did it do this with a previous version? Have you tried .17?

**Comment by AzGilrock** [comment]
I’d do an experiment and disable backups

**Comment by skier150** [comment]
I went to FILE -> PREFERENCES -> BACKUP, unchecked Backup On Launch and Backup Subfolders.  Unfortunately, it still freezes.


    On Sunday, October 29, 2023 at 07:50:54 PM EDT, Gil Jones ***@***.***> wrote:  
 
 


I’d do an experiment and disable backups

**Comment by derwin12** [comment]
I think Gil was referring to this setting .. perhaps set to disabled or something like 10mins and see if the problem moves to ever y 10mins. Normally this backup is seemless.
![image](https://github.com/smeighan/xLights/assets/4643499/9ee6791d-5949-42ca-bcc7-0e25165551be)

**Comment by skier150** [comment]
I am sorry, I missed this email.  I found that setting, and it had a direct affect on the freezing.  Whatever interval I set, the freezing followed at the same interval.  When I disabled it, the freezing stopped.

    On Friday, November 3, 2023 at 04:47:42 PM EDT, Daryl ***@***.***> wrote:  
 
 


I think Gil was referring to this setting .. perhaps set to disabled or something like 10mins and see if the problem moves to ever y 10mins. Normally this backup is seemless.
